CoroutineTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +13,8 @@
         android:networkSecurityConfig="@xml/network_security_config"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:replace="android:theme">
```

Comment:
Just curious, what's the reason we had to make this change? 🙏

Reply:
@Tuubz The project build fails without this change. I am finding solution for it, that's why the PR is masked as Draft 🙈